src/juju/actions.js

```diff
@@ -0,0 +1,12 @@
+// Action labels
+export const actionsList = {
+  updateModelList: "UPDATE_MODEL_LIST"
+};
+
+// Action creators
+export function updateModelList(models) {
```

Comment:
Should action creators be tested?

Reply:
Yep, there are [some examples](https://redux.js.org/recipes/writing-tests#example) in the docs.